package.xml

```diff
@@ -29,7 +29,7 @@
   <build_depend>rqt_gui</build_depend>
   <build_depend>rqt_gui_cpp</build_depend>
   <build_depend>sensor_msgs</build_depend>
-  
+  <depend>topic_tools</depend>
```

Comment:
Depend tags are often on top. Set of the same tag are always seperated by an empty line.